pkg/kudoctl/cmd/convert.go

```diff
@@ -0,0 +1,32 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/kudobuilder/kudo/pkg/kudoctl/cmd/convert"
+	"github.com/spf13/cobra"
+
+	"github.com/kudobuilder/kudo/pkg/kudoctl/util/vars"
+)
+
+// newConvertCmd creates a new command that shows the plans available for an instance
+func newConvertCmd() *cobra.Command {
```

Comment:
what do you think about `kudo package import ...` or `kudo package helm ...` is it possible that we'll ever have more inputs than just helm? I guess it's highly unlikely, right?

Reply:
I think we could have a CNAB convert as well.  I'm fine with:
```bash
kubectl kudo package import helm
kubectl kudo package import cnab
```